geomstats/geometry/special_euclidean.py

```diff
@@ -311,44 +312,45 @@ def compose(self, point_1, point_2, point_type=None):
         rotations = self.rotations
         dim_rotations = rotations.dimension
 
-        point_1 = self.regularize(point_1, point_type=point_type)
-        point_2 = self.regularize(point_2, point_type=point_type)
+        point_a = self.regularize(point_a, point_type=point_type)
+        point_b = self.regularize(point_b, point_type=point_type)
 
         if point_type == 'vector':
-            n_points_1, _ = point_1.shape
-            n_points_2, _ = point_2.shape
+            n_points_a, _ = point_a.shape
+            n_points_b, _ = point_b.shape
 
-            assert (point_1.shape == point_2.shape
-                    or n_points_1 == 1
-                    or n_points_2 == 1)
+            if not (point_a.shape == point_b.shape
+                    or n_points_a == 1
+                    or n_points_b == 1):
+                raise ValueError()
 
-            if n_points_1 == 1:
-                point_1 = gs.stack([point_1[0]] * n_points_2)
+            if n_points_a == 1:
+                point_a = gs.stack([point_a[0]] * n_points_b)
```

Comment:
I don't think this is necessary: `gs.matmul(point_a, point_b)` will distribute the multiplication of `point_b` to all points of `point_a` and conversely

Reply:
Indeed, thank you!